File: sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/Exceptions.scala
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
package com.azure.cosmos.spark

import com.azure.cosmos.CosmosException

private object Exceptions {
  def isResourceExistsException(cosmosException: CosmosException) : Boolean = {
    cosmosException.getStatusCode == CosmosConstants.StatusCodes.ConflictStatusCode
  }
}

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/builders/Exceptions.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.azure.monitor.opentelemetry.exporter.implementation.builders;

import java.util.List;

import static java.util.Collections.singletonList;

public final class Exceptions {

    public static List<ExceptionDetailBuilder> minimalParse(String str) {
        ExceptionDetailBuilder builder = new ExceptionDetailBuilder();
        builder.setStack(str);
        return singletonList(builder);
    }

    private Exceptions() {
    }
}
